### DISCUSSION TITLE
Expecting type ISODateTime

### DISCUSSION THREAD

**Discussion by Chillance** [shurcooL/graphql#88] [Q&A]
So, I'm getting this error message about this ISODateTime type being expected, but I can't see that in this library. How do I fix this?

Thanks!

**Comment by dmitshur** [comment]
Is that a custom scalar type that the GraphQL server you're connecting to defines in its schema?

If so, you'll need to define your own Go type accordingly.

See [here](https://github.com/shurcooL/githubv4/blob/ee671ab25ff03db0246cbd7f08fa0bc88a2f9c15/scalar.go#L29-L30) for an example of a [`DateTime`](https://godoc.org/github.com/shurcooL/githubv4#DateTime) Go type that was created for GitHub's [`DateTime`](https://developer.github.com/v4/scalar/datetime/) GraphQL scalar.

**Comment by marcocali** [comment]
As @dmitshur suggested you can create your own `type`:

`
type (
	ISODateTime struct{ time.Time }
)
`

Than use it in your `Query/Mutation/Variables`.
